mongoose.h

```diff
@@ -1605,7 +1605,11 @@ struct mg_dns {
 };
 
 struct mg_addr {
-  uint8_t ip[16];    // Holds IPv4 or IPv6 address, in network byte order
+  union {    // Holds IPv4 or IPv6 address, in network byte order
+    uint8_t ip[16];
+    uint32_t ip4;
+    uint64_t ip6[2];
```

Comment:
maybe uint32_t[4] ? easier to deal with

Reply:
Yeah, but that would mean 4 32-bit comparisons, it will only remove the 64-bit alignment constraint. Are you thinking on old compilers not being able to handle uint64_t ?